packages/portal/src/components/iiif/IIIFMiradorViewer.vue

```diff
@@ -44,6 +44,11 @@
         default: null
       },
 
+      webResources: {
```

Comment:
These aren't being passed in to the viewer. They are only needed for the swiper one level up.
No need for this prop.

Reply:
Indeed. It's gone.